moderator/settings.py

```diff
@@ -122,7 +121,7 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
 
 HMAC_KEYS = config('HMAC_KEYS', cast=json.loads)
 
-PASSWORD_HASHERS = get_password_hashers(BASE_PASSWORD_HASHERS, HMAC_KEYS)
+PASSWORD_HASHERS = get_password_hashers(PASSWORD_HASHERS, HMAC_KEYS)
```

Comment:
I don't think that we store password because of persona, but let's be consistent.